tests/framework/datavolume.go

```diff
@@ -0,0 +1,19 @@
+package framework
+
+import (
+	"kubevirt.io/containerized-data-importer/tests/utils"
+
+	cdiv1 "kubevirt.io/containerized-data-importer/pkg/apis/datavolumecontroller/v1alpha1"
+)
+
+func (f *Framework) CreateDataVolumeFromDefinition(dataVolume *cdiv1.DataVolume) (*cdiv1.DataVolume, error) {
+	return utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
+}
+
+func (f *Framework) DeleteDataVolume(dataVolume *cdiv1.DataVolume) error {
```

Comment:
As a small added value this could be changed to `CreateDataVolumeFromDefinitionOrDie` and not return _error_,  test the err return from `utils.CreateDataVolumeFromDefinition`. 
Are you and @awels in agreement on what separates code in _framework_ vs _utils_? So far, to me, it looks like _framework_ defines and instantiates a *Framework and defines convenience metrhods that call the same func in _utils_ passing some fields from *Framework.

Reply:
> Are you and @awels in agreement on what separates code in framework vs utils? 

@awels I'd be in favor of only adding functions to the framework package if the functions do something beyond simply calling something in the utils package directly.